src/Symfony/Component/EventDispatcher/composer.json

```diff
@@ -29,6 +29,10 @@
     "conflict": {
         "symfony/dependency-injection": "<3.4"
     },
+    "provide": {
+        "psr/event-dispatcher-implementation": "1.0",
```

Comment:
Can we do that (because this is only true when the `psr/event-dispatcher` package is present)?

Reply:
It still does provide the implementation - when the package is present of course. When it isn't, nothing can provide it by definition. So yes we can.